### PR TITLE
update cmd to get trust anchors for linkerd 2.9

### LINKED ISSUE #855: Instructions to get current trust anchors don't work for Linkerd 2.9
Documentation page: https://linkerd.io/2/tasks/manually-rotating-control-plane-tls-credentials/#bundle-your-original-trust-anchor-with-the-new-one

Section: Bundle your original trust anchor with the new one

The command to get the current trust anchors PEM doesn't work with Linkerd 2.9.

The current comand on the page is:

```
kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.global}' |  \
jq -r .identityContext.trustAnchorsPem > original-trust.crt
```

A possible replacement for Linkerd 2.9 is (this is using [yq](https://github.com/mikefarah/yq)):

```
kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}' | \
yq r - global.identityTrustAnchorsPEM > original-issuer.crt
```

### PR DESCRIPTION
**Pull request by frigus02**
Linkerd 2.9 changed the config map format. This means the command mentioned to get the current trust anchors PEM does not work anymore.

This adds command to the documentation, which works with the new config format.

Fixes #855

I wasn't sure if I should leave the command for Linkerd < 2.9 in there or replace it. Also not sure if `yq` is the best thing to use here. Happy to change to whatever you prefer.

### CODE REVIEW

**Comment by frigus02** [comment]
The build fails because of broken links, which don't seem to be broken. I guess some of the links return 500 every now and then?

**Review by Pothulapati — APPROVED** [review]
Thanks for the fix! :)

File: linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md [hunk 1]
```diff
@@ -117,12 +117,18 @@ certificates](/2/tasks/generate-certificates/).
...
 ```bash
-kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.global}' |  \
-jq -r .identityContext.trustAnchorsPem > original-trust.crt
+# For Linkerd < 2.9.0:
+kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.global}' \
+  | jq -r .identityContext.trustAnchorsPem > original-trust.crt
+
+# For Linkerd >= 2.9.0:
+kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}' \
+  | yq r - global.identityTrustAnchorsPEM > original-issuer.crt
```
[the remark this answers is not in the document]
Comment (frigus02) [reply linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md]: Ah yes, of course 🤦. Good catch.

[the remark this answers is not in the document]
Comment (frigus02) [reply linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md]: Fixed. Even the build is happy now 🎉

**Pull request merged**